USNRewardPayOut.sol

```diff
@@ -0,0 +1,32 @@
+import "./SampleOffer.sol";
+
+
+contract USNRewardPayOut {
+     
+     SampleOffer public usnContract;
+     
+     function USNRewardPayOut(SampleOffer _usnContract) {
+          usnContract = _usnContract;
+     }
+
+     // interface for USN
+    function payOneTimeReward() returns(bool) {
+        if (msg.value < usnContract.getDeploymentReward())
```

Comment:
Because:

1) The DAO needs to be able to vote and set those at any value it seems fit, so it needs to be in the offer contract which has to be in the whitelist. Keep in mind it is just a `SampleOffer` and you can and should modify/add/remove parameters for your own proposal as you see fit.

2) This contract `USNRewardPayOut` is separate and will not be in the whitelist and needs no integration with the DAO.